src/gui/static/src/app/components/pages/wallets/change-name/change-name.component.ts

```diff
@@ -13,42 +13,76 @@ import { SoftwareWalletService } from '../../../../services/wallet-operations/so
 import { WalletBase } from '../../../../services/wallet-operations/wallet-objects';
 import { HardwareWalletService } from '../../../../services/wallet-operations/hardware-wallet.service';
 
+/**
+ * States in which ChangeNameComponent can be.
+ */
 enum States {
-  Initial,
-  WaitingForConfirmation,
+  /**
+   * Showing the form.
+   */
+  Initial = 'Initial',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
  Initial = "Initial",
```